mne/channels/channels.py

```diff
@@ -1710,6 +1710,8 @@ def combine_channels(inst, groups, method='mean', keep_stim=False,
     elif isinstance(inst, BaseEpochs):
         combined_inst = EpochsArray(new_data, info, events=inst.events,
                                     tmin=inst.times[0])
+        if inst.metadata is not None:
+            combined_inst.metadata = inst.metadata
```

Comment:
```suggestion
            combined_inst.metadata = inst.metadata.copy()
```
I think it's safer.

can you also add a what's new entry? and update a test 🙏

Reply:
Yes, will do!